src/slices/AccountSlice.ts

```diff
@@ -147,7 +150,7 @@ export const getBalances = createAsyncThunk(
         fiatDaowsohm: ethers.utils.formatEther(fiatDaowsohmBalance),
         wsohmAsSohm: ethers.utils.formatUnits(wsohmAsSohm, "gwei"),
         pool: ethers.utils.formatUnits(poolBalance, "gwei"),
-        mockSohm: ethers.utils.formatUnits(mockSohmBalance, "gwei"), // delete before mainnet
+        mockSohm: ethers.utils.formatUnits(mockSohmBalance, "gwei"), // is there a way to include this ONLY on testnet?
```

Comment:
Seems like you could: https://amberley.dev/blog/2020-09-07-conditionally-add-to-array-or-obj/

Reply:
Why not do it like this:
```
let mockSohmBalance = null;
if (addresses[networkID].MOCK_SOHM) {
   // set values
}

return {
   balances: {
      // other values
      ...(mockSohmBalance && {mockSohm: ethers.utils.formatUnits(mockSohmBalance, "gwei")}),
   }
}
```

Reply:
perfect